configure

```diff
@@ -2336,6 +2339,9 @@ fi
 #
 echo_n "Checking endianness... "
 cat > tmp_endianness_check.cpp << EOF
+#ifndef __attribute__
```

Comment:
This doesn't work as __attribute__ is not defined as a macro.
You are basically voiding __attribute__((used)) here.

Reply:
Oops, indeed, sorry for that. Now using `__GNUC__` instead.

Reply:
Another way could be to use C++11 attribute specifiers but until C++17 it is not guaranteed that an unknown attribute would be ignored so this way is fine.